.github/dependabot.yml

```diff
@@ -8,9 +8,7 @@ updates:
   - package-ecosystem: "npm"
     directory: "/" # ToDo: check if this updates example-app/
     schedule:
-      interval: "weekly"
-      day: "monday"
-      time: "09:00"
+      interval: "monthly"
```

Comment:
Any particular reason for this?

Reply:
I want to propose monthly updates since the weekly updates are time consuming.